src/components/Header/Header.css

```diff
@@ -0,0 +1,18 @@
+.header {
+  width: 100%;
+  background-color: var(--g3-color__base-blue-light);
+  height: 100px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header__logo {
+  height: 80px;
+  width: 100px;
```

Comment:
The logo image is horizontally stretched, I think we should give "auto" to width here